internal/cmd/utils.go

```diff
@@ -59,6 +59,23 @@ func filterInstancesByRegion(instances []turso.Instance, region string) []turso.
 	return result
 }
 
+func probeLocation(location string) *time.Duration {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", "http://turso-bouncer.fly.dev:8080/", nil)
```

Comment:
The "official" URL should be `http://region.turso.io:8080`. My bad for giving you the internal one.